tools/rna_tools/pipmir/tool-data/twobit_update.loc.sample

```diff
@@ -0,0 +1,23 @@
+#This is a sample file distributed with Galaxy that is used by some
```

Comment:
now I update twobit.loc with 4 columns so I think it may be good to use a new name to differentiate, or...?